compel/Makefile

```diff
@@ -17,6 +17,10 @@ ldflags-y		+= -r
 # UAPI inclusion, referred as <compel/...>
 ccflags-y		+= -I compel/include/uapi
 
+ifeq ($(ARCH),mips)
+ccflags-y		+= -I compel
```

Comment:
Why is this required? Does MIPS port has different headers layout than other architectures?

Reply:
it is used to compile handler-elf.c. in compel/src/lib/handler-elf.c . I used this :
#ifdef CONFIG_MIPS
#include "arch/mips/src/lib/include/ldsodefs.h"
#endif